smarts/core/mission_planner.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         self, waypoints: Waypoints, road_network: SumoRoadNetwork, agent_behavior=None
     ):
         self._waypoints = waypoints
-        self._agent_behavior = agent_behavior or AgentBehavior(aggressiveness=5)
+        self._agent_behavior = agent_behavior
```

Comment:
This was a default in the case that someone applied a task in a scenario but did not match it with a specified aggressiveness. If there is not going to be a default here we may need an assertion when translating the agent spec and mission into an agent.

Reply:
Thanks, I added it back.